### PR TITLE
Module 1: Portfolio website design for large screens

### PR DESCRIPTION
**Pull request by aimalamiri**
### Desktop version of the page

1. Designed the navbar for large screens.
2. Designed the header for large screens.
3. Designed the middle section of the page for large screens.
4. Designed the cards inside the middle section of the page for large screens.
5. Designed the about me section for large screens.
6. Designed the contact me section for large screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,25 +83,29 @@ <h2 class="header-subtitle text-white font-inter">
         <h2 class="works-title text-black font-crete-round">
           My Recent Works
         </h2>
-        <hr />
+        <div>
```
Comment (BenMukebo) [inline index.html:86]: - [ ] Great job, congratulation, you have done a great job, I personally appreciate your work 💪🏻 , I think we can make this better by adding some few things, on hover action for each card, please make those texts on the image display none and make the see project display flex, you designed it already but not in a good way, let me show you with an image,
hover
😄  hope this makes sense, so please implement a hover action make those text and capacity display:none and display flex the see project button

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@aimalamiri,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Design looks awesome ✔️ 
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [x] Kindly, Add your project image to the README file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (aimalamiri) [reply index.html:86]: Hi, I'm planing to add these functionalities in my next task

**Review by ghost — APPROVED** [review]
# STATUS: APPROVED :tada:  :confetti_ball: :fireworks: 
Hi  @aimalamiri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**